src/app/pages/admin/twins/details/twins.details.component.ts

```diff
@@ -192,15 +199,15 @@ export class TwinsDetailsComponent implements OnInit, OnDestroy {
       },
     };
     this.twinsService.editTwin(twin).subscribe(
-      resp => {
+      _resp => {
```

Comment:
There is a reason for this `_resp`?

Reply:
Yes, it's a style standard to mark of the unused arguments.

Reply:
See here https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html#flag-unused-declarations-with---nounusedparameters-and---nounusedlocals

Reply:
I'm not against but if you only fix it in a few places it's not consistent. Rather create an issue and fix it latter